lib/controller.js

```diff
@@ -980,6 +983,41 @@ controller.tesselEnvVersions = opts => {
   });
 };
 
+controller.createNewProject = init.createNewProject;
+
+controller.crashReporter = function(options) {
+  var cr = Promise.resolve();
+
+  // t2 crash-reporter --on
+  if (options.on) {
+    cr = CrashReporter.on();
+  } else if (options.off) {
+    // t2 crash-reporter --off
+    cr = CrashReporter.off();
+  }
+
+  // t2 crash-reporter --test
+  if (options.test) {
+    // not handling failures, as we want to trigger a crash
+    cr.then(CrashReporter.test)
+      .then(module.exports.closeSuccessfulCommand);
+  } else {
+    cr.then(CrashReporter.status)
+      .then(
+        module.exports.closeSuccessfulCommand,
+        module.exports.closeFailedCommand
+      );
+  }
+
+  return cr;
+};
+
+controller.installDrivers = function() {
+  return drivers.install();
```

Comment:
I removed the `then(...)` here, because that's handled by `callControllerWith(...)` in `bin/tessel-2.js`